### PR TITLE
itest-config: add provider channel and juju channel configs

### PR DESCRIPTION
**Pull request by kelkawi-a**
itest-config: add provider channel and juju channel configs to integration test workflow (example [here](https://github.com/charmed-kubernetes/actions-operator#multiple-controllers)). This will enable the operator environment setup to be more dynamic according to the developer's needs.

### CODE REVIEW

**Review by arturo-seijas — COMMENTED** [review]
Can you update the ìntegration_tests section of the readme as well?

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by yanksyoon — COMMENTED** [review]
Thanks for your contribution! Just a minor comment to think about :)

File: .github/workflows/integration_test_run.yaml [hunk 1]
```diff
@@ -83,6 +83,14 @@ on:
         description: Actions operator provider as per https://github.com/charmed-kubernetes/actions-operator#usage
         type: string
         default: microk8s
+      channel:
+        description: Actions operator provider channel as per https://github.com/charmed-kubernetes/actions-operator#usage
+        type: string
+        default: 1.26-strict/stable
```
[the remark this answers is not in the document]
Comment (kelkawi-a) [reply .github/workflows/integration_test_run.yaml]: Changed default to 1.26/stable. Thank you!

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by yanksyoon — APPROVED** [review]
LGTM :)

**Comment by jdkandersson** [comment]
Thanks, @weiiwang01 the failure seems to be related to the rockcraft test?

**Comment by weiiwang01** [comment]
> Thanks, @weiiwang01 the failure seems to be related to the rockcraft test?

Yes, the failure doesn't seem to relate to the changes in the PR, I will do some test on that.

**Comment by weiiwang01** [comment]
> > Thanks, @weiiwang01 the failure seems to be related to the rockcraft test?
> 
> Yes, the failure doesn't seem to relate to the changes in the PR, I will do some test on that.

Hi @kelkawi-a , the problem in workflow self checks has been fixed, thank you for helping us found the problem.

Could you please update your PR branch with the latest change? This should rerun all checks and after all checks passed, you can merge this pull request.

**Pull request merged**